cmd/ocm-load-test.go

```diff
@@ -173,23 +174,18 @@ func run(cmd *cobra.Command, args []string) error {
 		os.Exit(1)
 	}
 
-	if viper.GetString("ocm-token") == "" {
-		logger.Fatal(cmd.Context(), "ocm-token is a necessary configuration")
+	if viper.Sub("ocm") == nil {
+		logger.Fatal(cmd.Context(), "ocm is a necessary configuration")
 	}
 	err = helpers.CreateFolder(cmd.Context(), viper.GetString("output-path"), logger)
 	if err != nil {
 		logger.Fatal(cmd.Context(), "creating api connection: %v", err)
 	}
 	logger.Info(cmd.Context(), "Using output directory: %s", viper.GetString("output-path"))
 
-	connection, err := helpers.BuildConnection(cmd.Context(), viper.GetString("gateway-url"),
-		viper.GetString("client.id"),
-		viper.GetString("client.secret"),
-		viper.GetString("ocm-token"),
-		logger,
-	)
+	connections, err := ocm.BuildConnections(cmd.Context(), logger)
 	if err != nil {
-		logger.Fatal(cmd.Context(), "creating api connection: %v", err)
+		return err
```

Comment:
Should there still be a log message around the fact that it errored?

Reply:
It is still happening, but it was moved to the new `BuildConnections` method, here we are just receiving the error from there.